Automattic/wp-calypso
# Fix/migrate enzyme to react testing library

### Quick summary
The button-group component is using the enzyme framework for testing, We need to migrate it to React-testing-library and jest

### Testing instructions
1. Run the test for button-group block with yarn run test-client:watch from source directory
2. Select P and then type button-group
3. We should see that the correct test file was executed and run like this:
![](https://d.pr/i/LuxMS4+)
[Direct Link](https://d.pr/i/LuxMS4+)

Related to #63370

Fixes #63370.
